### DISCUSSION TITLE
How to count repositories in Organizations

### DISCUSSION THREAD

**Discussion by zhou-hao** [anuraghazra/github-readme-stats#1914] [Q&A] [answered]
Most of the repository are in my organization. I want to count them together.

**Comment by rickstaa** [comment, marked as answer]
@zhou-hao Thanks for your question. Including data from organizations is our most requested feature (see #1). A PR is open for supporting this feature, but we have not yet merged it. It can, however, already be used on your own Vercel instance by following [these steps](https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174).
